learn/advanced/updating.md

```diff
@@ -1,30 +1,33 @@
 # Update to the latest Meilisearch version
 
-Currently, Meilisearch databases can only be opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to **migrate an existing database** from an older version of Meilisearch to the most recent one.
+:::warning
+
+1. This guide does not work for versions below v0.15. For more information, contact [support](https://slack.meilisearch.com/).
+2. If you're using v0.24 or below, use the `X-MEILI-API-KEY: apiKey` authorization header:
+
+ <CodeSamples id="updating_guide_check_version_old_authorization_header" />
+:::
+
+Currently, Meilisearch databases can only be opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to migrate an existing database from an older version of Meilisearch to the most recent one.
 
 If you already have a Meilisearch database with some data you don’t want to lose, you are in the right place!
 
 :::note
-If you have already **installed the latest version and manually indexed your data and settings**, you can ignore this guide.
+If you have already installed the latest version and manually indexed your data and settings, you can ignore this guide.
 :::
```

Comment:
Do we really need this note? Isn't it essentially saying "if you have already updated meilisearch, you don't need to follow this guide on updating meilisearch"?